tests/src/test/java/alluxio/shell/AbstractAlluxioShellTest.java

```diff
@@ -93,6 +93,16 @@ protected void copyToLocalWithBytes(int bytes) throws IOException {
     Assert.assertEquals(getCommandOutput(new String[] {"copyToLocal", "/testFile",
         mLocalAlluxioCluster.getAlluxioHome() + "/testFile"}), mOutput.toString());
     fileReadTest("/testFile", 10);
+    mOutput.reset();
```

Comment:
@aaudiber  Sure, I will write a new method to test the relative path under `copyToLocalCommandTest`.